vector/src/main/java/im/vector/fragments/VectorRoomDetailsMembersFragment.java

```diff
@@ -772,6 +774,8 @@ public void onClick(View v) {
                 intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_MATRIX_ID, mSession.getMyUserId());
                 intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_ROOM_ID, mRoom.getRoomId());
                 intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_ADD_CONFIRMATION_DIALOG, true);
+                intent.putExtra(VectorRoomCreationActivity.EXTRA_ROOM_CREATION_ACTIVITY_MODE, VectorRoomCreationActivity.RoomCreationModes.DISCUSSION);
```

Comment:
Note: the value DISCUSSION is correctly use here

Reply:
Finally EXTRA_ROOM_CREATION_ACTIVITY_MODE can be remove, this is the default value